test/libyul/objectCompiler/metadata.yul

```diff
@@ -1,11 +1,16 @@
 object "A" {
   code {
-    pop(datasize("x"))
-    pop(datasize("C"))
+    let x := datasize("x")
+    let c := datasize("C")
+    mstore(0, x)
```

Comment:
I fear mstores like that might be optimized away soon, too.

Reply:
I'll use `sstore` then.

Reply:
can you add `return` instructions like you did in the other tests?

Reply:
Done.

Reply:
> can you add return instructions like you did in the other tests?

Looks like I missed this comment. Should I add them in a new PR?

Reply:
It's fine, we can do it when we encounter it.